getting-started/setup-multi-node-basic.md

```diff
@@ -0,0 +1,111 @@
+point for user access; 
+- One or more data nodes to store and organize distributed data.     
+
+All nodes begin as single-node TimescaleDBs, i.e., hosts with a running 
+PostgreSQL instance and a loaded TimescaleDB extension.  More detail on 
+the architecture can be found [here][architecture]. For the below setup 
+instructions, "data node" and "access node" will be used to refer to both the 
+server running TimescaleDB, as well as the TimescaleDB instance itself.
+The steps for creating a basic multi-node setup are as follows:
+
+## Basic setup
+1. Configure data nodes for communication with the access node
+1. Initialize data nodes from the access node 
+
+### Prerequisites
+- One PostgreSQL instance to act as an access node
+- One or more PostgreSQL instances to act as data nodes
+- TimescaleDB [installed][install] and [set up][setup] on all nodes
+- Access to a superuser role (e.g. `postgres`) on all nodes
+
+### 1. Configure data nodes for node-to-node communication
+To enable communication between the access node and the data node(s), the
+data node must be configured to authorize connections from the access node.  The 
+simplest way is to use "trust" authentication which provides unencumbered client 
+access to the nodes. This method must be applied to every data node.
+
+>:WARNING: The "trust" authentication method allows insecure access to all 
+nodes.  For production implementations, please use more secure 
+methods of authentication (see [advanced setup][] for examples).
+
+#### Edit authentication configuration file on data nodes
+Client authentication is usually configured in the `pg_hba.conf`([reference doc]
+[postgresql-hba]) file located in the data directory.  If the file is not located 
+there, access the instance (psql) and execute the command:
+
+```sql
+SHOW hba_file;
+``` 
+
+To enable "trust" authentication, add a line to `pg_hba.conf` to allow the
+access to the instance:
+
+```
+# TYPE  DATABASE  USER  ADDRESS         METHOD
+host    all       all   {access_node_ip}   trust
+```
+
+where `{access_node_ip}` is the ip address for the access node.
+
+#### Edit main configuration
+It is necessary to change the parameter `max_prepared_transactions` to a 
+non-zero value if it hasn't been changed already ('150' is recommended). The 
+parameter is located in `postgresql.conf`, typically in the data directory. If it 
+isn't there, access the node (psql) and get the path with:
+
+```
+SHOW config_file;
+```
+
+##### Reload server configuration
+
+To reload the server configuration, you can use the following command
+on the data node:
+
+```
+pg_ctl reload
+```
+
+### 2. Initialize data nodes from the access node
+From within the access node (psql), use the command:
+
+```sql
+SELECT add_data_node('{node_name}', host => '{data_node_address}');
+```
+
+`{node_name}` should be a unique name for the node. `{data_node_address}` 
+is the host name or IP address of the data node.
```

Comment:
Since you have to repeat the replaceable anyway, it might be better to use real examples. Easier to follow.